### DISCUSSION TITLE
Uncaught ReferenceError: XMLHttpRequest is not defined in chrome extension webworker in the sdk version "^10.4.0"

### DISCUSSION THREAD

**Discussion by chirag-chhajed** [firebase/firebase-js-sdk#7687] [Q&A]
Firebase SDK version: "^10.4.0"

https://github.com/firebase/firebase-js-sdk/issues/6124#issuecomment-1092889054

I am making a chrome extension with firebase. I have set up my firebase in the webworker. I am trying to add a doc to the firestore and it gives an error that XMLHttpRequest is not defined in webworker which is fine. But in a firebase-js-sdk issue it gives the a solution that use firebase sdk version 9 or above because it uses fetch instead of XMLHttpRequest but I am already using "^10.4.0" then why I am getting this error if I am using latest version of sdk.

![image](https://github.com/firebase/firebase-js-sdk/assets/69468696/ea3e58e2-8a1a-49ad-9685-3fcb0864f276)

**Comment by hammadkhan** [comment]
I am facing the same problem? Did you find any solution yet? Thanks
